source/lib-intro.tex

```diff
@@ -478,13 +478,14 @@
 implementation-defined types\iref{type.descriptions},
 and member functions\iref{functions.within.classes}.
 
-\rSec3[expos.only.func]{Exposition-only functions}
+\rSec3[expos.only.entity]{Exposition-only entities, etc.}
 
 \pnum
-Several function templates
+Several entities and \grammarterm{typedef-name}{s}
 defined in \ref{\firstlibchapter} through \ref{\lastlibchapter} and \ref{depr}
 are only defined for the purpose of exposition.
-The declaration of such a function is followed by a comment ending in \expos.
+The declaration of such an entity and \grammarterm{typedef-name}
```

Comment:
```suggestion
The declaration of such an entity or \grammarterm{typedef-name}
```

Reply:
This is from LWG3818 which supposedly was not intended to be part of the paper.

Reply:
I think it was intended to be. See ~3 comments starting from https://github.com/cplusplus/draft/pull/5982#issuecomment-1320991610.